includes/obstacles.h

```diff
@@ -61,7 +66,7 @@ class ObstacleHolder : public Obstacle
 
         if (!textureImage.loadFromFile(ObstacleHolder<T>::textureFilename))
         {        
-            std::cout << "Texture file " + ObstacleHolder<T>::textureFilename + ".png could not be loaded.";
+            std::cout << "Texture file " + ObstacleHolder<T>::textureFilename + " could not be loaded.";
```

Comment:
I prefer outputting to cerr rather than cout in error cases. Actually, in production you would want to use a more robust logging framework like spdlog but for a side project its fine I guess.